help/actions/_posts/2015-02-09-export.md

```diff
@@ -140,10 +148,22 @@ The output field specifiers are added to the beginning of the custom format as d
 
     This is a special format directive. It must be the first command specified in a format string, and there can be multiple *%Z* directives at the beginning of the string. While the *Z* tag produces no output, it allows you to specify the value of certain variables with corresponding parameters. It has the following syntax: **%Zcommand:parameter**. The parameter can be a number, a word, or a string, depending on the command. If it is a string, it must be enclosed in double quotes. The available commands are: 
 
-    * **Encoding:word** This specifies the encoding of the output. Available encoding parameters are: 
-        * **unicode** (default): UTF-8 encoding of all unicode characters. 
+    There are two encoding options: global and field. As the name suggests specifying global encoding applies the encoding to the entire citiation, all output field. Please see further down for explanation of field encoding.
+    * **Encoding:word** Available global encoding parameters are: 
+        * **unicode** (default): UTF-8 encoding of all unicode characters. Note also for backward compatibility **UTF-8** option for word is also supported.
         * **html**: This encoding converts characters in URLs to hex notation, and & to &amp;. Outside URLs, the characters &, <, >, and " are converted to &amp;, &lt;, &gt;, and &quot;, respectively. 
```

Comment:
* github pages is converting your html entities into the actual characters; wrap `&amp;`, `&lt;`, `&gt;`, and `&quot;` in backticks so they don't convert

Reply:
done.